openmdao/core/system.py

```diff
@@ -282,8 +283,8 @@ def pre_setup(self, problem):
         User-configurable method to be run when problem.setup() is called
         but prior to any actual problem setup.
 
-        Args
-        ----
+        Parameters
+        ----------
```

Comment:
I thought we were using the keyword `Args` here to prevent confusion with `params`... Keith did some tweaking to Docs to enable this?